mmv1/products/integrations/AuthConfig.yaml

```diff
@@ -0,0 +1,470 @@
+          - decryptedCredential.0.service_account_credentials
+          - decryptedCredential.0.oidc_token
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'type'
+            description: |
+              Authentication type, e.g. "Basic", "Bearer", etc.
+          - !ruby/object:Api::Type::String
+            name: 'token'
+            description: |
+              The token for the auth type.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'serviceAccountCredentials'
+        description: |
+          Service account credential.
+        conflicts:
+          - decryptedCredential.0.username_and_password
+          - decryptedCredential.0.oauth2_authorization_code
+          - decryptedCredential.0.oauth2_client_credentials
+          - decryptedCredential.0.jwt
+          - decryptedCredential.0.auth_token
+          - decryptedCredential.0.oidc_token
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'serviceAccount'
+            description: |
+              Name of the service account that has the permission to make the request.
+          - !ruby/object:Api::Type::String
+            name: 'scope'
+            description: |
+              A space-delimited list of requested scope permissions.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'oidcToken'
+        description: |
+          Google OIDC ID Token.
+        conflicts:
+          - decryptedCredential.0.username_and_password
+          - decryptedCredential.0.oauth2_authorization_code
+          - decryptedCredential.0.oauth2_client_credentials
+          - decryptedCredential.0.jwt
+          - decryptedCredential.0.auth_token
+          - decryptedCredential.0.service_account_credentials
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'serviceAccountEmail'
+            description: |
+              The service account email to be used as the identity for the token.
+          - !ruby/object:Api::Type::String
+            name: 'audience'
+            description: |
+              Audience to be used when generating OIDC token. The audience claim identifies the recipients that the JWT is intended for.
+          - !ruby/object:Api::Type::String
+            name: 'token'
+            description: |
+              ID token obtained for the service account.
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'tokenExpireTime'
+            description: |
+              The approximate time until the token retrieved is valid.
+
+              A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+            output: true
+  - !ruby/object:Api::Type::NestedObject
+    name: 'client_certificate'
+    description: |
+      Raw client certificate
+    ignore_read: true
```

Comment:
Note to self (just resolve this): We use `ignore_read` here over `url_param_only` because we access the value in `pre_create`. We could use `url_param_only` but it would require a longer `pre_create`.